task/script/0.1/tests/test_local.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# only used for local quick testing, since this project has its own test run strategy described in CONTRIBUTING.md
+# USAGE: cd into directory of this script, then execute: ./run_local.sh
+
+set -ex
+
```

Comment:
this seems to be good. Can you please move this outside of tests, maybe in a different directory named as `support`. Also you can mention about this script file in README.md so that if somebody wants to use this then they can use this. Btw to test tasks locally we already have a script in tests directory which you can trigger by running
```sh
bash test/run-test.sh <task-name> <version>
```

Reply:
Thanks! I move the script and documented both scripts.

PS: In between I thought about merging them together, but ran into quite some troubles and gave up on it (actually I deleted a namespace that I did not intent to), so I guess it is least safer this way.